flink-connector-postgres-cdc/src/main/java/com/ververica/cdc/connectors/postgres/source/fetch/PostgresScanFetchTask.java

```diff
@@ -158,17 +166,19 @@ private void executeBackfillTask(
 
         // we should only capture events for the current table,
         // otherwise, we may not find corresponding schema
-        PostgresSourceConfig pgSourceConfig = (PostgresSourceConfig) ctx.getSourceConfig();
+        PostgresSourceConfig config = (PostgresSourceConfig) ctx.getSourceConfig();
         Configuration dbzConf =
                 ctx.getDbzConnectorConfig()
                         .getConfig()
                         .edit()
                         .with("table.include.list", split.getTableId().toString())
                         .with(
-                                "slot.name",
-                                pgSourceConfig.getDbzProperties().getProperty("slot.name")
+                                SLOT_NAME.name(),
+                                config.getDbzProperties().getProperty(SLOT_NAME.name())
                                         + "_"
-                                        + pgSourceConfig.getSubtaskId())
+                                        + config.getSubtaskId())
```

Comment:
Could we offer a static util method like `getSlotnameForBackfillTask()` in PostgresSourceConfig ?